### ISSUE TITLE
Intermediate results during training

### ISSUE THREAD

**Issue by unlugi** [LeeDoYup/DeblurGAN-tf#19]
![GOPR0384_11_00_blur_000066](https://user-images.githubusercontent.com/15035152/54817249-81a78680-4ca7-11e9-9c6c-c1558945a5cb.png)

![GOPR0396_11_00_blur_000053](https://user-images.githubusercontent.com/15035152/54817268-8ff5a280-4ca7-11e9-91ce-7c7a4bce4620.png)

I am training the network from scratch. Are the results above meaningful for epoch 65 on the test set? I will continue training until 300 epochs.

**Comment by xianfengju** [comment]
I trained the network from scratch, have not got a ideal result. Did you get the ideal result？

**Comment by LeeDoYup** [comment]
I got satisfactory result, but i will check some bug.
If you find some thing, please make pull requests.

**Comment by xianfengju** [comment]
ok，thanks.Did you set batchSize=1?

**Comment by LeeDoYup** [comment]
Yes i set the batchsize=1, i changed readme.txt about this issue.
